redisenterprise/datadog_checks/redisenterprise/check.py

```diff
@@ -0,0 +1,322 @@
+import sys
+from datetime import datetime, timedelta
+
+import requests
```

Comment:
The usage of requests directly is not recommended. Please use `self.http.get()` instead. This will take care of things like proxies, kerberos authentication etc. You can read more in https://datadoghq.dev/integrations-core/base/http/
You should also includes `template: init_config/http` and `instances/http` in your configuration spec

Reply:
@hithwen  I've removed the use of requests and spec.yml  now includes  the http templates